.circleci/config.yml

```diff
@@ -97,7 +97,7 @@ jobs:
       - run: *run_yarn_install
       - save_cache: *save_yarn_cache
       - build_app:
-          template: 'Autocomplete.js'
+          template: 'Autocomplete.js 0'
```

Comment:
Maybe "v0" or "v0.x" would be more explicit.

Reply:
We use this pattern for other templates too. Let's rename all of them in a major?

Reply:
Sure!